lib/ansible/modules/net_tools/haproxy.py

```diff
@@ -33,90 +31,99 @@
   backend:
     description:
       - Name of the HAProxy backend pool.
-    default: auto-detected
+      - If this parameter is unset, it will be auto-detected.
+    type: str
   drain:
     description:
       - Wait until the server has no active connections or until the timeout
-        determined by wait_interval and wait_retries is reached.  Continue only
-        after the status changes to 'MAINT'.  This overrides the
-        shutdown_sessions option.
+        determined by wait_interval and wait_retries is reached.
+      - Continue only after the status changes to 'MAINT'.
+      - This overrides the shutdown_sessions option.
     type: bool
     version_added: "2.4"
   host:
     description:
       - Name of the backend host to change.
+    type: str
     required: true
   shutdown_sessions:
     description:
       - When disabling a server, immediately terminate all the sessions attached
-        to the specified server. This can be used to terminate long-running
-        sessions after a server is put into maintenance mode. Overridden by the
-        drain option.
+        to the specified server.
+      - This can be used to terminate long-running sessions after a server is put
+        into maintenance mode. Overridden by the drain option.
     type: bool
-    default: 'no'
+    default: no
   socket:
     description:
       - Path to the HAProxy socket file.
+    type: path
     default: /var/run/haproxy.sock
   state:
     description:
       - Desired state of the provided backend host.
-      - Note that C(drain) state was added in version 2.4. It is supported only by HAProxy version 1.5 or later,
-        if used on versions < 1.5, it will be ignored.
+      - Note that C(drain) state was added in version 2.4.
+      - It is supported only by HAProxy version 1.5 or later,
+      - When used on versions < 1.5, it will be ignored.
```

Comment:
```suggestion
      - When used on earlier versions of HAProxy, it will be ignored.
```

Reply:
This is not exactly the same. The exact version from which it was ignored is lost.